client/pages/restaurant-page.jsx

```diff
@@ -0,0 +1,167 @@
+      rating = <>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star-half rated"></i>
+      </>;
+    } else if (selectedRestaurant.rating === 3) {
+      rating = <>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+      </>;
+    } else if (selectedRestaurant.rating === 2.5) {
+      rating = <>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star-half rated"></i>
+      </>;
+    } else if (selectedRestaurant.rating === 2) {
+      rating = <>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star rated"></i>
+      </>;
+    } else if (selectedRestaurant.rating === 1.5) {
+      rating = <>
+        <i className="fas fa-star rated"></i>
+        <i className="fas fa-star-half rated"></i>
+      </>;
+    } else if (selectedRestaurant.rating === 1) {
+      rating = <>
+        <i className="fas fa-star rated"></i>
+      </>;
+    }
+
+    return (
+      <>
+        <Container style={{
+          height: '475px',
+          backgroundImage: `url(${selectedRestaurant.image_url})`,
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat'
+        }} className='d-flex flex-column-reverse shadow'>
+          <Container>
+          <h3 className='d-flex primary-color-font text-nowrap mb-0 text-shadow'>{selectedRestaurant.name}</h3>
+            <div>{rating}</div>
+            <p className='d-flex primary-color-font text-nowrap flex-column-reverse mb-0'>{`${selectedRestaurant.review_count} Reviews`}</p>
+            <p className='primary-color-font text-white font-italic mb-2'>{`${categories.join(', ')}`}</p>
+          </Container>
+        </Container>
+        <Container>
+          <Row className='d-flex justify-content-center'>
+          <h2 className='text-center primary-color pt-1'>Location & Hours</h2>
+          </Row>
+          <Row className='border-top border-danger primary-font' style={{ marginBottom: '4rem' }}>
+            <Column className='border border-danger rounded mt-4 mr-3 shadow'>
+            <h3 className='mt-3'>{selectedRestaurant.name}</h3>
+            <p className='mb-0'>{selectedRestaurant.location.address1}</p>
+            <p className='mb-0'>{selectedRestaurant.location.display_address[zipIndex]}</p>
+            </Column>
+            <Column className='border border-danger rounded mt-4 ml-3 shadow' style={{ fontSize: '.9rem' }}>
+              <p className='dflex flex-row mt-3 mb-1'>{hoursByDay[0].day} <span className='float-right'>{hoursByDay[0].hours}</span></p>
+              <p className='dflex flex-row my-1'>{hoursByDay[1].day} <span className='float-right'>{hoursByDay[1].hours}</span></p>
```

Comment:
Your `dflex` isn't working because it's actually `d-flex`. Then you can `justify-conent-end`.